internal/generator/testdata/components/components/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - configmap.yaml
+components:
+  - componentA
```

Comment:
I dunno if this directory should be committed? It isn't required for the test nor is `configmap.yaml`, but both is required if you want to run kustomize in the directory (e.g. `kubectl kustomize`).

Reply:
If this not used in tests then why commit it?

Reply:
I have removed the files not needed for the test.